specificatie/genereervariant/openapi.json

```diff
@@ -2593,8 +2618,8 @@
               "minItems": 2,
               "type": "array",
               "example": [
-                196733.51,
-                439931.89
+                "196733.510",
+                "439931.890"
               ],
               "deprecated": true,
```

Comment:
Deze moet niet meer depricated zijn toch?

Reply:
Goeie vraag, in develop is ie deprecated in master niet, hoe kan dat? 
Deze branch is afgetakt van develop.
Voorstel was toch om uiteindelijk een bbox te gebruiken waarbij coördinaat linksonder = coördinaat rechtsboven? Als dat nog de bedoeling is, dan is deprecated terecht.

Reply:
In versie 1.2 is de locatie-parameter bij panden aanwezig. We hebben vast bij het bedenken van de contour toevoegingen een moment gehad waarop het een goed idee was om locatie deprecated te maken. 
Met de huidige inzichten is deprecated maken blijkbaar niet meer nodig dus dat kunnen we dan terugdraaien.